test/E2ETest/Microsoft.Test.E2E.AspNet.OData/ComplexTypeInheritance/ComplexTypeInheritanceTests.cs

```diff
@@ -48,6 +48,116 @@ public ComplexTypeInheritanceTests(WebHostTestFixture fixture)
             }
         }
 
+        public static TheoryDataSet<string, string, string,bool> NewCollectionMembers
```

Comment:
>NewCollectionMembers [](start = 65, length = 20)

rename it meaningful

Reply:
Does PostToCollectionNewComplextTypeMembers seem good?